src/Geometry/main.ts

```diff
@@ -0,0 +1,27 @@
+import { Circle } from './Circle';
+import { Rectangle } from './Rectangle';
+import { Square } from './Square';
+import { Triangle } from './Triangle';
+import { Trapezoid } from './Trapezoid';
+function main() {
```

Comment:
do not spam bunch of main file. How if we just create relevant method with specific name like `geo_test()`, `car_test()`, etc..
Then having only one main file includes above functions.